src/streamlink/plugins/ert.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+
+import re
+import requests
+
+from streamlink.plugin import Plugin
+from streamlink.plugin.api import http, validate, useragents
+
+YOUTUBE_URL = "https://www.youtube.com/watch?v={0}"
+req = requests.get('http://extreme-ip-lookup.com/json/', params={'User-Agent': useragents.CHROME})
```

Comment:
I dislike this random website call, it should be done without this.

Reply:
I am using it purely for locating user's countrycode, nothing more.

Reply:
This will be run on import, which is not a good idea. Can this lookup be avoided as well?

Reply:
Even original website does this lookup via javascript to display certain iframe, geolocked stream. It can be avoided with a try-except block by prioritizing greek geo-locked stream but it will have significant overhead... i.e. trying playing the stream only to fail due to Youtube's denial.

Reply:
If you are going to call that IP lookup service, it must be done on-demand. 

In the case of geo-blocked videos, the YouTube will fail before returning any streams and fairly quickly. I agree with @back-to that the IP lookup should be avoided, if that service goes away or doesn't allow connections from proxy servers then the plugin may fail.

Reply:
some other solution

---

use the IP lookup but use the logger with a **warning** about it
and use it somewhere in `class Ert(Plugin):` not at the top of the ert.py file

**cache the countrycode** for 30 days or more

---

use a command for the  countrycode `--ert-country XX`

Reply:
I am not sure how to do that.